components/heap/heap_trace_standalone.c

```diff
@@ -10,6 +10,10 @@
 #include "esp_heap_trace.h"
 #undef HEAP_TRACE_SRCFILE
 
+//Todo: fix #include "esp_spi_flash.h"
+#include <stdbool.h>
+bool spi_flash_cache_enabled(void);
```

Comment:
The build complains it cannot find this file. Where do I look to fix that?

Reply:
@chipweinberger modify CMakeLists.txt to add spi_flash as a dependency.

Reply:
thanks. got it.